src/makeCalculator.js

```diff
@@ -4,7 +4,35 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(num) {
+      this.result += num;
+    },
+    subtract(num) {
+      this.result -= num;
+    },
+    multiply(num) {
+      this.result *= num;
+    },
+    divide(num) {
+      if (num !== 0) {
+        this.result /= num;
+      }
```

Comment:
Consider throwing an error or returning a specific value when attempting to divide by zero. Currently, the divide method silently fails without providing feedback, which might not be the expected behavior for the user of the calculator.